server/utils/s3.js

```diff
@@ -20,20 +20,24 @@ export const makePolicy = () => {
       ['content-length-range', 0, +process.env.AWS_S3_UPLOAD_MAX_SIZE],
       ['starts-with', '$Content-Type', 'image'],
       ['starts-with', '$Cache-Control', ''],
+      { 'x-amz-algorithm': 'AWS4-HMAC-SHA256'},
+      { 'x-amz-credential': credential },
+      { 'x-amz-date': longDate },
     ],
     expiration: format(tomorrow, 'YYYY-MM-DDTHH:mm:ss\\Z'),
   };
 
   return new Buffer(JSON.stringify(policy)).toString('base64');
 };
 
-export const signPolicy = (policy: any) => {
-  invariant(AWS_SECRET_ACCESS_KEY, 'AWS_SECRET_ACCESS_KEY not set');
-  const signature = crypto
-    .createHmac('sha1', AWS_SECRET_ACCESS_KEY)
-    .update(policy)
-    .digest('base64');
+export const getSignature = (policy: any) => {
+  const kSecret = "AWS4" + AWS_SECRET_ACCESS_KEY;
```

Comment:
The inbuild node `crypto` module has support for these hashing methods, so we should use that rather than rely on a third-party npm module that also isn't maintained